compass/ocean/tests/spherical_harmonic_transform/qu_convergence/__init__.py

```diff
@@ -0,0 +1,67 @@
+from compass.testcase import TestCase
+
+from compass.ocean.tests.spherical_harmonic_transform.qu_convergence.mesh \
+     import Mesh
+from compass.ocean.tests.spherical_harmonic_transform.qu_convergence.init \
+     import Init
+# from compass.ocean.tests.spherical_harmonic_transform.qu_convergence.analysis \
+#      import Analysis
+
+
+class QuConvergence(TestCase):
```

Comment:
Sorry, I was fine with QUConvergence, too.  My comment was about the docstring not the class name. But this is fine, too.

Reply:
I'm fine either way. So I'll just leave it like this.